<?xml version="1.0"?>
<launch>

  <arg name="motor_count" default="2" />
  <arg name="joint_counts" default="[3,3]" />

  <param name="motor_count" value="$(arg motor_count)" />
  <rosparam param="joint_counts"  subst_value="True">$(arg joint_counts)</rosparam>

  <node pkg="bata_base" type="bata_interface.py" name="bata_interface" output="screen"/>

</launch>
